parsers/TW.py

```diff
@@ -9,10 +10,7 @@ def fetch_production(country_code='TW', session=None):
     r = session or requests.session()
     url = 'http://data.taipower.com.tw/opendata01/apply/file/d006001/001.txt'
     response = requests.get(url)
-    content = response.content
```

Comment:
This is safe to do?  Can you run this parser and check it still works.

Reply:
I do not know how to run the parser but perhaps we should have Travis CI run this test on every pull request.